deployment/entity-service/values.yaml

```diff
@@ -129,6 +129,10 @@ api:
         cpu: 250m
         memory: 256Mi
 
+    ## Annotations added to the database init job's pod.
+    # podAnnotations: {}
```

Comment:
I thought those brackets shouldn't be there in a yaml config.

Reply:
I see them fairly regularly to indicate whether the expected type is a map or a list.